packages/hardhat-zksync-solc/src/compile/index.ts

```diff
@@ -42,16 +43,32 @@ export async function compile(zksolcConfig: ZkSolcConfig, input: CompilerInput,
     return await compiler.compile(input, zksolcConfig);
 }
 
+export async function link(zksolcConfig: ZkSolcConfig, linkLibraries: LinkLibraries) {
+    let compiler: ICompiler;
+
+    if (zksolcConfig.compilerSource === 'binary') {
+        compiler = new BinaryCompiler('');
```

Comment:
I am not sure what it is about and where empty strings are used, but seems irrelevant to the linker functionality.